packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3463,11 +3450,41 @@ export class ContainerRuntime
 				...tagCodeArtifacts({
 					id: toPath,
 					fromId: fromPath,
+					pkg: packagePath?.join("/"),
+					fromPkg: fromPackagePath?.join("/"),
 				}),
 			});
 			return;
 		}
-		this.garbageCollector.addedOutboundReference(fromPath, toPath, timestampMs);
+		this.garbageCollector.addedOutboundReference({
+			fromNodePath: fromPath,
+			toNodePath: toPath,
+			packagePath,
+			fromPackagePath,
+			timestampMs,
+		});
+	}
+
+	/**
+	 * Called to retrieve the package path of the node with the given path. The node should be root or belong to a
+	 * data store or an attachment blob.
+	 */
+	public getGCNodePackagePath(nodePath: string): readonly string[] | undefined {
+		// GC uses "/" when adding "root" references, e.g. for Aliasing or as part of Tombstone Auto-Recovery.
```

Comment:
Why did you move this?

Reply:
Just to make it next to where it's used.